ui/inspectors/system-section.reel/overview.reel/overview.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * @module ui/inspectors/system-section.reel/overview.reel
+ */
+var Component = require("montage/ui/component").Component,
+    Model = require("core/model/model").Model;
+
+/**
+ * @class Overview
+ * @extends Component
+ */
+exports.Overview = Component.specialize(/** @lends Overview# */ {
+
+    systemInfo: {
+        value: null
+    },
+
+    templateDidLoad: {
+        value: function () {
+            var self = this,
+                loadingPromises = [];
+            this.isLoading = true;
+            this.systemInfo = {};
```

Comment:
It may be more readable to extract `systemInfo` building in its own method of `system-info-service` which could take care of building, caching and refreshing the object when needed.